tests/conftest.py

```diff
@@ -107,39 +108,38 @@ def request_mock_exception_batch_creation(
     return mock
 
 
-@pytest.fixture(scope="function")
+@pytest.fixture()
```

Comment:
Is this change related to ruff?

Reply:
yes, PT family rule: https://docs.astral.sh/ruff/rules/#flake8-pytest-style-pt